spec/requests/api/v1/users/sign_up_spec.rb

```diff
@@ -5,15 +5,24 @@
 
   context 'when attributes are good' do
     let(:user_email) { Faker::Internet.email }
+    let(:password) { Faker::Internet.password }
+    let(:confirm_success_url) { Faker::Internet.url }
     let(:user_params) do
-      { email: user_email, password: Faker::Internet.password }
+      { email: user_email, password: password, password_confirmation: password }
     end
 
     it 'creates an user account' do
       expect(response).to have_http_status(:created)
       expect(User.count).to be(1)
       expect(User.find_by(email: user_email)).to_not be_nil
     end
+
+    it 'sends an email for confirmation' do
+      mails = ActionMailer::Base.deliveries
+
+      expect(mails.size).to eq(1)
+      expect(mails.first.to.first).to eq(user_email)
```

Comment:
We should split in two separate examples, `it 'sends an email'` and `it 'the email is sent to the correct email'`